qualang_tools/__init__.py

```diff
@@ -7,6 +7,7 @@
     "results",
     "plot",
     "analysis",
+    "macros",

```

Comment:
@yomach, is there anything preventing this from being in the "addons" folder?

Reply:
No, we can put it there.
We need to think on the structure, it seems that currently almost everything is in it's one file :)

Reply:
I definitely agree and I also don't like having one folder for each tool because it makes it hard for customers to find them :)

Reply:
Hopefully it's a good place for people to put macros in the future. Happy to change the structure later